### ISSUE TITLE
MacOS - Unison - Fatal error: Server: Fatal error during unmarshaling

### ISSUE THREAD

**Issue by JParkinson1991** [EugenMayer/docker-sync#688]
### Error
```
UNISON 2.51.2 (OCAML 4.06.1) finished propagating changes at 08:13:54.10 on 23 Oct 2019
Saving synchronizer state
Synchronization complete at 08:13:54  (1 item transferred, 0 skipped, 0 failed)

...

Unison 2.51.2 (ocaml 4.08.1): Contacting server...
Connected [//a/local/file/path -> //c7ed5a9a65f7//app_sync]
Looking for changes
  Waiting for changes from server
Reconciling changes
Fatal error: Server: Fatal error during unmarshaling (input_value: ill-formed message),
possibly because client and server have been compiled with differentversions of the OCaml compiler.
```
It seems the unsion sync strategy is no longer working for MacOS using the latest brew version of unison. `brew install unison`

My assumption is the unsion binary/package used by docker-sync internals has been compiled using OCAML 4.06.1 but the version of unison available via the brew package manager has been compiled using OCAML 4.08.1. See brew package dependencies here: https://formulae.brew.sh/formula/unison

I have tried using https://opam.ocaml.org/ to compile my own version of unison with OCAML 4.06.1 but was getting absolutely nowhere.

Im not sure how but i feel like we need to bump up the internal unsion OCAML version so that people can still use the brew available unison package. Another option would be to provide details or a method of compiling/downloading a compatible unison version.

Note that native_osx syn strategy works 'absolutely' fine, i quote absolutely due to the stability issues of using OSXFS.

### Docker Driver
Docker For Mac

### Sync strategy
unison 

_No error when using native_osx, assumed due to using macs inbuilt unison binary compiled using ocaml 4.06.1?_

### docker-sync.yml
```
version: '2'

options:
    compose-file-path: 'docker-compose.yml'
    compose-dev-file-path: 'docker-compose.override.yml'
    project_root: config_path

syncs:
    app-sync:
        monit_enable: true
        notify_terminal: true
        src: './app'
        sync_strategy: unison
        sync_userid: 82
        #sync_groupid: 82
```

### OS
MacOS 10.15

**Comment by JParkinson1991** [comment]
Sorry, just seen the other issue, can be closed as duplicate of #685

**Comment by JParkinson1991** [comment]
Aplogies, noticed this is a duplicate of #687
